installer/home/pom.xml

```diff
@@ -1,9 +1,7 @@
-<project
```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional update buried in the noise. Please revert these updates.